docker/snippets/app.conf

```diff
@@ -0,0 +1,13 @@
+
+location / {
+  send_timeout 10m;
+  include snippets/cors.conf;
+
+  client_max_body_size 100M;
+
+  passenger_app_env production;
+  passenger_enabled on;
+  passenger_app_root /srv/app;
+
+  passenger_env_var APP_ENV production;
```

Comment:
This `APP_ENV=production` is gonna be okay on Travis, right?

Reply:
It worked fine on my Travis tests but your question is interesting. Thing is Nginx overrides all environment variable and Passenger set it to production but I can pass the variable in the docker compose.

Reply:
Yeah it is better if if have one central location for setting env variables and the rest of the code uses it as needed.